udata/migrations/2020-06-11-add-resource-fs-filename.py

```diff
@@ -0,0 +1,41 @@
+'''
+The purpose here is to fill every resource with a fs_filename string field.
+'''
+import logging
+from urllib.parse import urlparse
+
+from udata.models import Dataset, CommunityResource
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
+    log.info('Processing resources resources.')
+
+    datasets = Dataset.objects()
```

Comment:
```suggestion
    datasets = Dataset.objects().no_cache()
```

Using this may avoid the current migration failure on demo. Use it on every query set below too.